env-TEST/lang-ENG/data/storybook-learning-events.csv

```diff
@@ -0,0 +1,26 @@
+id,time,android_id,package_name,storybook_id,learning_event_type
```

Comment:
Trying to understand better here, What is the thought process behind columns chosen in storybook-learning-events.csv ?For instance, I didn't get how we can benefit from id,android_id,package_name ?

Reply:
@Keerthi4308 These columns represent the learning event data that is collected by the analytics app: https://github.com/elimu-ai/analytics/blob/master/app/src/main/java/ai/elimu/analytics/receiver/StoryBookLearningEventReceiver.java

- `android_id` is the ID of the Android device (representing a student's ID).
- `package_name` is the name of the Android app that the student used to read the storybook (e.g. [Vitabu](https://github.com/elimu-ai/vitabu)).

Feel free to suggest other names of the columns if you have any ideas to make the data more readable 🙂

---

Next step will be to preprocess the data so that it can be used for training a machine learning model. For example counting the number of time a child has opened a book, how many pages the child read, etc.

Reply:
Thanks for explaining. Looks good to me.